### PR TITLE
Update cellDuration description in Scheduler article

### PR DESCRIPTION
**Pull request by czernous-devexpress**

### CODE REVIEW

File: api-reference/10 UI Widgets/dxScheduler/1 Configuration/cellDuration.md [hunk 1]
```diff
@@ -5,6 +5,6 @@ default: 30
 ---
 ---
 ##### shortDescription
-Specifies cell duration in minutes.
+Specifies cell duration in minutes. The interval between [startDayHour]({basewidgetpath}/Configuration/#startDayHour) and [endDayHour]({basewidgetpath}/Configuration/#endDayHour) should be divisible by this value.
```
Comment (AlexJameson) [inline api-reference/10 UI Widgets/dxScheduler/1 Configuration/cellDuration.md]: Hello. Please do not use {basewidgetpath} for links in doc members that belong to a certain UI component. In this case, replace them with links that lead to the Scheduler startDayHour and endDayHour properties.

Comment (czernous-devexpress) [reply api-reference/10 UI Widgets/dxScheduler/1 Configuration/cellDuration.md]: done

File: api-reference/10 UI Widgets/dxScheduler/1 Configuration/cellDuration.md [hunk 2]
```diff
@@ -5,6 +5,6 @@ default: 30
 ---
 ---
 ##### shortDescription
-Specifies cell duration in minutes.
+Specifies cell duration in minutes. The interval between [startDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#endDayHour) should be divisible by this value.
```
Comment (AlexJameson) [inline api-reference/10 UI Widgets/dxScheduler/1 Configuration/cellDuration.md]: I also suggest you to get rid of passive voice as follows:
```suggestion
Specifies cell duration in minutes. This property's value should divide the interval between [startDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#endDayHour) into even parts.
```

**Review by RomanTsukanov — APPROVED** [review]

**Review by AlexJameson — APPROVED** [review]

File: api-reference/10 UI Widgets/dxScheduler/1 Configuration/cellDuration.md [hunk 3]
```diff
@@ -5,6 +5,6 @@ default: 30
 ---
 ---
 ##### shortDescription
-Specifies cell duration in minutes.
+Specifies cell duration in minutes. This property's value should divide the interval between [startDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#endDayHour) into even parts.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Widgets/dxScheduler/1 Configuration/cellDuration.md:8]:
```suggestion
Specifies cell duration in minutes. This property's value should evenly divide the interval between [startDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#endDayHour).
```

Comment (dirk-pieterse) [reply api-reference/10 UI Widgets/dxScheduler/1 Configuration/cellDuration.md:8]: What do you mean by "cell duration"? Can you explain?

**Review by dirk-pieterse — APPROVED** [review]

File: api-reference/10 UI Widgets/dxScheduler/1 Configuration/cellDuration.md (hunk 3 above)
Comment (czernous-devexpress) [reply api-reference/10 UI Widgets/dxScheduler/1 Configuration/cellDuration.md:8]: Cell duration denotes the amount in minutes allocated per cell. This value affects the visual representation of cells within the Scheduler.

Comment (dirk-pieterse) [reply api-reference/10 UI Widgets/dxScheduler/1 Configuration/cellDuration.md:8]: Ok

**Pull request merged**